interfaces/apparmor/spec_test.go

```diff
@@ -139,6 +141,11 @@ func (s *specSuite) TestAddUpdateNS(c *C) {
 	s.spec.AddUpdateNS("s-u-n snippet 1")
 	s.spec.AddUpdateNS("s-u-n snippet 2")
 
+	// Check the order of the snippets can be retrieved.
+	idx, ok := s.spec.UpdateNSIndexOf("s-u-n snippet 2")
```

Comment:
do we need this test to run `s.spec.AddUpdateNS("s-u-n snippet 2")` again to show that it doesn't get put in multiple times?

Reply:
I can add it though it feels like testing the internal implementation. I'm perhaps biased because this is still fresh in my head.